### ISSUE TITLE
Table/mobile experience

### ISSUE THREAD

**Issue by ssunday** [cynicaloptimist/improved-initiative#463]
Just opening up an issue as there's some 'weirdness' with running it on say an iPad mini (my case). I'm going to play with it to find some specific things to improve. I think most of it's just responsive styling, but others could be tap/click related.

**Comment by cynicaloptimist** [comment]
Yeah, CSS and Mobile-focused PRs are always welcome, those are some of my weak spots.

**Comment by cynicaloptimist** [comment]
I'm open to them, here are some guidelines:

- Open a proposal as a new issue before doing significant work
- Smaller, more frequent PRs are strongly preferred
- Including a screenshot helps a lot

Take a look at the discussion on this PR for some context. I wound up having to do a huge amount of work after the fact to clean up some of the changes that weren't quite right. https://github.com/cynicaloptimist/improved-initiative/pull/476

**Comment by pspeter3** [comment]
Thank you for thoughtful comment. I will look through it and try and make a demo for it first.
